src/runtime/graph/debug/graph_runtime_debug.cc

```diff
@@ -0,0 +1,133 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file graph_runtime_debug.cc
+ */
+#include <tvm/runtime/packed_func.h>
+#include <tvm/runtime/registry.h>
+#include <tvm/runtime/ndarray.h>
+#include <chrono>
+#include "../graph_runtime.h"
+
+namespace tvm {
+namespace runtime {
+
+/*!
+ * \brief Graph runtime with debug .
+ *
+ *  This is the extension of GraphRuntime class used for debugging
+ *  TVM runtime PackedFunc API.
+ */
+class GraphRuntimeDebug : public GraphRuntime {
+ public:
+    /*!
+     * \brief Run each operation and get the output.
+     * \param index The index of op which needs to be run.
+     */
+  double DebugRun(int index) {
+    auto tbegin = std::chrono::high_resolution_clock::now();
+    if (op_execs()[index]) {
+      op_execs()[index]();
+    }
```

Comment:
For gpu, we should sync stream by calling this function on ctx
https://github.com/dmlc/tvm/blob/a03c60ba1c83d39b857b827f144bf1b371d0b0c7/python/tvm/_ffi/runtime_ctypes.py#L200

Reply:
@merrymercy  I invoked sync from python. I got the issue with that. The timestamp won't reflect if I invoked from python. I will update. Thanks.